.skiff/webapp.jsonnet

```diff
@@ -596,37 +590,81 @@ function(
         ]
     );
     
-    local olmo3_32b_instruct_worker = curriedCreateWorker(
-        'olmo-3-0625-32b-instruct',
-        env,
-        sharedEnv,
+    // local olmo3_32b_instruct_worker = curriedCreateWorker(
```

Comment:
We don't need instruct just yet and it was causing problems so I'm spinning it down for now. Once the model is up we can spin this back up.